Tools/AutoRest.Settings.targets

```diff
@@ -90,40 +89,46 @@
     </PropertyGroup>
     <Message Text="Copying assembly files to $(_CodeSignDirectory)." />
     <Message Text="Copying symbols and documentation files to $(_BinariesDirectory)." />
-    <MakeDir Directories="$(_CodeSignDirectory)" Condition="!Exists($(_CodeSignDirectory))" />
+    <MakeDir Directories="$(_CodeSignDirectory)" Condition="!Exists('$(_CodeSignDirectory)')" />
     <ItemGroup>
       <_AutoRestBinaries Include="$(ProjectDir)$(OutputPath)*.*"/>
     </ItemGroup>
 
-    <Copy SourceFiles="$(ProjectDir)$(OutputPath)$(AssemblyName).xml;$(ProjectDir)$(OutputPath)$(AssemblyName).pdb"
-          DestinationFolder="$(_BinariesDirectory)"/>
-    <Copy SourceFiles="$(ProjectDir)$(OutputPath)$(AssemblyName).dll" 
+    <Copy Condition=" '$(OS)' == 'Windows_NT'" ContinueOnError="True"
```

Comment:
It was laziness do to the way that pdbs are named in mono, mdb. AutoRest.exe.mdb doesn't fit, so I just set it to continue rather than fail. I'll trim this down and make it more specific.